sympy/printing/latex.py

```diff
@@ -2279,7 +2287,8 @@ def latex(expr, **settings):
 
     long_frac_ratio: The allowed ratio of the width of the numerator to the
     width of the denominator before we start breaking off long fractions.
-    The default value is 2.
+    The default value is width of the numerator to the width of the
```

Comment:
This is difficult to parse: it says that the default value of "The allowed ratio of the width of the numerator to the width of the denominator before we start breaking off long fractions" is "width of the numerator to the width of the denominator in the expression itself". How about this: 

"If None (the default value), long fractions are not broken up."